SelectionShort Java/SelectionShort.java

```diff
@@ -0,0 +1,28 @@
+
+package SelectionShort;
+
+public class SelectionShort {
+    public static void main(String[] args) {
+        
+        int [] number = {8,3,4,7,2,6,5,1,10,9};
+        Selection_Short(number);
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp4enaadiy/SelectionShort Java/SelectionShort.java
+++ b/tmp/tmp4enaadiy/SelectionShort Java/SelectionShort.java
@@ -12,7 +12,7 @@
             int shortest = number[i];
                 for (int j=i ; j<number.length ; j++){
                     if (number[j] <= number[i]){
-                        number [i] = number [j];   
+                        number [i] = number [j];
                         number[j] = shortest;
                         shortest = number[i];
                     }
```